deploy/charts/cert-manager/signkey_annotation.txt

```diff
@@ -0,0 +1,2 @@
+fingerprint: 1020CF3C033D4F35BAE1C19E1226061C665DF13E
+url: https://cert-manager.io/public-keys/cert-manager-keyring-2021-09-20-1020CF3C033D4F35BAE1C19E1226061C665DF13E.gpg
```

Comment:
note: as mentioned in the PR description this isn't ideal, but it can easily be updated later